source/clear-linux/concepts/concepts.rst

```diff
@@ -16,3 +17,4 @@ different.
    restart
    security
    telemetry-about
+   debug
```

Comment:
@puneetse. I'd like this document to appear in tooling--so that it has more visibility--and so it fits future evolution of content in the guides.  In this case, just remove revisions to `concepts.rst`.  Then move `debug.rst` to: 
clear-linux-documentation/source/clear-linux/guides/tooling

Note: You don't need to make an entry in TOC of `guides.rst` because we use the `:glob:`, or wildcard, to get all.

Reply:
@mvincerx  sounds good. done.

Reply:
@puneetse Will you please assure that you moved `debug.rst` to the tooling directory? Also given that you have 1 image, please create subdirectory `debug` in which you should also place figure 1.  In testing this PR, location of debug.rst didn't appear to change.

Reply:
Yes, the document is moved under tooling. I had forgotten about the figure and moved that under the tooling/figures directory too.

I didn't understand what you mean by having the single image in subdirectory? Do you mean tooling/figures/debug/debug-diagram.png?

Reply:
You corrected it now. It shows a clean build, with the image rendering correctly.